builtin_string.go

```diff
@@ -157,6 +157,20 @@ func (r *Runtime) string_raw(call FunctionCall) Value {
 	}
 }
 
+func (r *Runtime) stringproto_at(call FunctionCall) Value {
+	r.checkObjectCoercible(call.This)
+	s := call.This.toString()
+	pos := call.Argument(0).ToInteger()
+	length := int64(s.length())
+	if pos < 0 {
+		pos = length + pos
+	}
+	if pos >= length || pos < 0 {
+		return _undefined
+	}
+	return newStringValue(string(s.charAt(toIntStrict(pos))))
```

Comment:
```suggestion
	return s.substring(int(pos), int(pos+1))
```
This will fix the UTF-16 issue. Same change is needed for `charAt`.

Reply:
just added these changes